+nosnoc/+solver/MpccSolver.m

```diff
@@ -198,14 +200,16 @@
         end
 
         function out = cat(dim,varargin)
-            error('Concatenation not supported')
+            error('Concatenation not supported.')
         end
 
         function varargout = size(obj,varargin)
             varargout = 1;
         end
         
         function ind = end(obj,k,n)
+            % TODO end is a matlab keyword, maybe use a different name?
```

Comment:
This is a required overload for overloading paren indexing. In cases where concatenation is not supported returning 1 is correct.

Reply:
please add this comment to the code